client/src/containers/scenarios/component.tsx

```diff
@@ -4,9 +4,12 @@ import { PlusIcon, XCircleIcon } from '@heroicons/react/solid';
 import ScenariosFilters from 'containers/scenarios/filters';
 import ScenariosList from 'containers/scenarios/list';
 import { AnchorLink } from 'components/button';
+import Lottie from 'lottie-react';
 import { useScenarios } from 'hooks/scenarios';
 import type { Page } from 'components/breadcrumb/types';
 
+import newScenarios from 'utils/lottie/new-scenario.json';
```

Comment:
could you move it to `client/src/containers/scenarios`? It's not an "util" and I think it's only for scenarios. Also if you can change the name to `noScenariosAnimationData` or something like that, please.